### ISSUE TITLE
area validation

### ISSUE THREAD

**Issue by beto-Sibileau** [UNICEFECAR/dash#96]
- Poverty (absolute poverty, image below), issue: countries appearing that doesn't report data
![image](https://user-images.githubusercontent.com/62593695/135245603-79379594-874f-47d9-9c8c-2b991845283d.png)

**Comment by lbassil** [comment]
Those countries are reporting a value of 0... if we decide to get rid of 0 values, this will improve the chart... but we need to discuss with the team for such a decision!

![image](https://user-images.githubusercontent.com/86417315/135257622-104d8096-543d-467a-a91d-16cb9ae64e38.png)

**Comment by beto-Sibileau** [comment]
thanks @lbassil  it looks correct to me and it's ok to report zero employed population for that indicator. I'm seeing the plot correctly now and can't reproduce the scenario where it happened so let's close the issue for now.

**Comment by lbassil** [comment]
This issue is still happening... I tried to fix it but so far couldnt know the reason of the weird behavior... it happens when we add the sorting: fig.update_xaxes(categoryorder="total descending")

**Comment by lbassil** [comment]
![GraphSorting](https://user-images.githubusercontent.com/86417315/135287343-b0a92bb5-aafc-4d8e-b07a-f47590ed7167.gif)

**Comment by beto-Sibileau** [comment]
thanks @lbassil. What I see from your animation is that the `xaxes` **country list category** is like the union resulting from the two callbacks displayed (even if there's no data in the last `df` displayed).
This looks to me that the attribute `xaxes` or whatever name it has in **plotly** is shared across callbacks and would therefore need to be somehow reset before doing the `update_xaxes` *category order*
What are your thoughts about it @jamescw

**Comment by jamescw** [comment]
@lbassil is this issue still open?

**Comment by lbassil** [comment]
Yeah we still didn't figure this out however it is not consistent and we cannot always replicate.

**Comment by beto-Sibileau** [comment]
hi guys, I linked this issue to a new pull request that I opened (with simple updates).

It looks to me that if we pass a `sort_values` at the data request level and suppress the `xaxis: category_order` we could circumvent this issue. What do you think @lbassil
